sds_data_manager/constructs/ialirt_processing_construct.py

```diff
@@ -179,19 +146,17 @@ def add_compute_resources(self):
         )
 
         # Specifies the networking mode as AWS_VPC.
-        # ECS tasks in AWS_VPC mode can be registered with
-        # Network Load Balancers (NLB).
         task_definition = ecs.Ec2TaskDefinition(
             self,
             "IalirtTaskDef",
-            network_mode=ecs.NetworkMode.AWS_VPC,
+            network_mode=ecs.NetworkMode.HOST,
```

Comment:
Can you add a quick comment about this change and why it needs to be HOST vs AWS_VPC?